frontend/src/services/proprietaryService.ts

```diff
@@ -41,8 +38,43 @@ export function useProprietaryService(
     const eliVal = toValue(eli)
     if (!eliVal || !dateAsString.value) return INVALID_URL
 
-    return `/norms/${toValue(eli)}/proprietary/${dateAsString.value}`
+    return `/norms/${eliVal}/proprietary/${dateAsString.value}`
   })
 
-  return useApiFetch<Proprietary>(url, { ...fetchOptions }).json()
+  return useApiFetch<Proprietary>(url, fetchOptions)
+}
+
+/**
+ * Convenience shorthand for `useProprietaryService` that sets the correct
+ * configuration for getting JSON data.
+ *
+ * @param eli ELI of the norm
+ * @param options Optional additional filters and queries
+ * @param [fetchOptions={}] Optional configuration for fetch behavior
+ * @returns Reactive fetch wrapper
+ */
+export const useGetProprietary: typeof useProprietaryService = (
+  eli,
+  options,
+  fetchOptions,
+) => useProprietaryService(eli, options, fetchOptions).json()
+
+/**
+ * Convenience shorthand for `useProprietaryService` that sets the correct
+ * configuration for putting JSON data.
+ *
+ * @param eli ELI of the norm
+ * @param options Optional additional filters and queries
+ * @param [fetchOptions={}] Optional configuration for fetch behavior
+ * @returns Reactive fetch wrapper
+ */
+export function usePutProprietary(
+  updateData: MaybeRefOrGetter<Proprietary | null>,
+  eli: Parameters<typeof useProprietaryService>["0"],
+  options: Parameters<typeof useProprietaryService>["1"],
+  fetchOptions: Parameters<typeof useProprietaryService>["2"],
+): ReturnType<typeof useProprietaryService> {
+  return useProprietaryService(eli, options, fetchOptions)
```

Comment:
i would expect this to set `{ refetch: false }` so there is a lower risk of forgetting to do this. But we can discuss that.

Reply:
this would also mean that the fetchOptions do not need to be set by the views as they are already "correctly" defined

Reply:
I would prefer if we could stick to the `useFetch` defaults. The reason being that I would like all services to behave the same, i.e. we should avoid having some services refetch automatically whereas others don't. Sticking with the `useFetch` default behavior seems to be the easiest way to achieve that, even if it is a bit more verbose. Would that be fine with you?

(At least for the services that is, I don't mind if composables set the refetch)

Reply:
I would have expected all gets to have `refetch` set to `true` and all other endpoints to have `immediate` set to `false` (the only exception to this so far is the render endpoint as it is a post endpoint but acts more like a get). But I'm also fine with having this in the composable layer.